src/usePostPatchPut.js

```diff
@@ -0,0 +1,25 @@
+import { useState, useCallback } from 'react';
```

Comment:
Is this file a duplicate?

Reply:
@arrested-developer  I added usePost.js hook but I didn't use it in the end. so it's the that is redundant. I forgot to delete it